src/pages/home/components/feature-description/feature-explanation-mobile.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { spaces, breakpoints, colors, textStyles } from '../../../../styles'
+import styled from 'styled-components'
+import Anchor from "../../../../components/anchor"
+
+const Headline = styled.h3`
+  margin: ${spaces.relax}px ${spaces.comfortable}px;
+
+  text-align: left;
+
+  ${textStyles.h3Bold}
+
+  @media screen and (min-width: ${breakpoints.desktop}px) {
+    margin: 0 
+    ${textStyles.h2Bold}
+  }
+`
+
+const Checklist = styled.img`
+  width: ${spaces.relax}px;
+  height: ${spaces.relax}px;
+  align-self: center;
+  margin-right: ${spaces.cozy}px;
+`
+
+const MyDiv = styled.div`
+  display: flex;
+  margin: 0 ${spaces.comfortable}px ${spaces.comfortable}px ${spaces.comfortable}px;
+`
+
+const Text = styled.div`
+${textStyles.h5}
+`
+
+
+class FeatureExplanationMobile extends Component {
+  render() {
+    return (
+      <div style={{display: "flex", flexDirection:"column"}}>
+        <img src={this.props.image} ></img>
+        <Headline>{this.props.title}</Headline>
+        <MyDiv>
+        <Checklist src="https://res.cloudinary.com/apomarn/image/upload/v1559056569/WeDo/checklist.png"></Checklist>
```

Comment:
use `map` here too, to render all the items of the checklist

and instead of receiving `text1`, `text2` and `text3`, you can receive an array of strings and use map to render them.

Reply:
I dont know how, can you teach me? 🙈